.vib/dokuwiki/goss/dokuwiki.yaml

```diff
@@ -0,0 +1,45 @@
+command:
+
+file:
+  /opt/bitnami/apache/conf/vhosts/dokuwiki-vhost.conf:
+    exists: true
+    filetype: file
+    contains:
+      - /DocumentRoot.*/opt/bitnami/dokuwiki/
+      - "RewriteEngine on"
+      - "RewriteRule ^_media/(.*)              lib/exe/fetch.php?media=$1  [QSA,L]"
+      - "RewriteRule ^_detail/(.*)             lib/exe/detail.php?media=$1  [QSA,L]"
+      - "RewriteRule ^_export/([^/]+)/(.*)     doku.php?do=export_$1&id=$2  [QSA,L]"
+      - "RewriteRule ^$                        doku.php  [L]"
+      - "RewriteCond %{REQUEST_FILENAME}       !-f"
+      - "RewriteCond %{REQUEST_FILENAME}       !-d"
+      - "RewriteCond %{REQUEST_URI}            !^/server-status$"
+      - "RewriteRule (.*)                      doku.php?id=$1  [QSA,L]"
+      - "RewriteRule ^index.php$               doku.php"
```

Comment:
This is okay, but I personally would not check the whole range of lines added (just a couple or the most relevant ones). We can assume that the rest of the lines will be added as well. WDYT?

Reply:
Sounds good to me. Do you think the following would cover the most relevant?
```
      - /DocumentRoot.*/opt/bitnami/dokuwiki/
      - "RewriteEngine on
      - "RewriteCond %{REQUEST_URI}            !^/server-status$"
      - "RewriteRule ^index.php$               doku.php"
```

Reply:
Yes, totally 😁